domain-models-runtime/src/test/java/org/folio/DemoRamlRestTest.java

```diff
@@ -345,6 +348,17 @@ public void invalidCqlClosesConnection(TestContext context) {
     }
   }
 
+  @Test
+  public void testForm(TestContext context) {
+    JsonPath jsonPath = given().spec(tenant)
+        .formParam("form1name", "form1value")
+        .formParam("form2name", "form2value")
+        .post("/rmbtests/testForm")
+        .then()
+        .statusCode(200).extract().body().jsonPath();
+    context.assertEquals("form1value", jsonPath.getString("[0].form1name"));
+    context.assertEquals("form2value", jsonPath.getString("[1].form2name"));
```

Comment:
```
        .then()
        .statusCode(200)
        .body("[0].form1name", is("form1value"),
              "[1].form2name", is("form2value"));
```
Rest-assured has built-in JSON path testing.

Reply:
Thx @julianladisch .. I new there was a better way!